### DISCUSSION TITLE
osgearth_imgui.exe imgui multiple screen problem

### DISCUSSION THREAD

**Discussion by yunusyurtturk** [pelicanmapping/osgearth#2660] [Support Q&A] [answered]
Hi, I updated my code to osgEarth-3.7 from a previous version after formatting my computer. 
After the update, I wasn't able to see ImGui panels in my application, then I tried osgearth_imgui.exe if it works. I realized that ImGui panels are not also rendered.  

I'm using 2 monitor setup.
The problem fixed after I add:
`viewer.setUpViewOnSingleScreen();`

Now they are rendered. You may want to check.

**Comment by gwaldron** [comment, marked as answer]
I pushed a patch for this in c6b4e56b4. Thanks.
